docs/csharp/misc/cs2022.md

```diff
@@ -12,4 +12,4 @@ ms.assetid: d22de497-c4ef-466f-8fbc-6faba7ba5ad0
 
 Options '/out' and '/target' must appear before source file names  
   
- The [**-out **(Set Output Filename)](../language-reference/compiler-options/output.md#outputassembly) and [-target (Specify Output File Format)](../language-reference/compiler-options/output.md#targettype) compiler options, when specified on the command line, must precede the source code files.
+ The [/out (Set Output Filename)](../language-reference/compiler-options/output.md#outputassembly) and [/target (Specify Output File Format)](../language-reference/compiler-options/output.md#targettype) compiler options, when specified on the command line, must precede the source code files.
```

Comment:
The docs settled on using `-` instead of `/` for compiler options. The `**` was meant to make it bold, but the extra space was an issue. I think the fix should either be:

```suggestion
 The [**-out** (Set Output Filename)](../language-reference/compiler-options/output.md#outputassembly) and [-target (Specify Output File Format)](../language-reference/compiler-options/output.md#targettype) compiler options, when specified on the command line, must precede the source code files.
```

or

```suggestion
 The [`-out` (Set Output Filename)](../language-reference/compiler-options/output.md#outputassembly) and [`-target` (Specify Output File Format)](../language-reference/compiler-options/output.md#targettype) compiler options, when specified on the command line, must precede the source code files.
```